doc/dev/devices.rst

```diff
@@ -0,0 +1,87 @@
+        border-bottom: #d3d3d3 1px solid;
+    }
+    .row-odd {
+        background-color: #f7f7f7;
+    }
+    </style>
+
+Supported devices
+=================
+
+Not all PennyLane devices currently work with Catalyst, and for those that do, their supported
+feature set may not necessarily match supported features when used without :func:`~.qjit`.
+
+Supported backend devices include:
+
+.. list-table::
+  :widths: 20 80
+
+  * - ``lightning.qubit``
+
+    - A fast state-vector qubit simulator written with a C++ backend. See the
+      `Lightning documentation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_qubit/device.html>`__
+      for more details, as well as its
+      `Catalyst configuration file <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/lightning_qubit/lightning_qubit.toml>`__.
+
+  * - ``lightning.kokkos``
+
+    - A fast state-vector qubit simulator utilizing the Kokkos library for CPU and GPU accelerated
+      circuit simulation. See the
+      `Lightning documentation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_kokkos/device.html>`__
+      for more details, as well as its
+      `Catalyst configuration file <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/lightning_kokkos/lightning_kokkos.toml>`__.
+
+  * - ``braket.aws.qubit``
+
+    - Interact with quantum computing hardware devices and simulators through Amazon Braket. To use
+      this device with Catalyst, make sure to install the
+      `PennyLane-Braket plugin <https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/>`__.
+      See the
+      `Catalyst configuration file <https://github.com/PennyLaneAI/catalyst/blob/main/runtime/lib/backend/openqasm/braket_aws_qubit.toml>`__
+      for supported operations.
+
+  * - ``braket.local.qubit``
+
+    - Run gate-based circuits on the Braket SDK's local simulator. To use
+      this device with Catalyst, make sure to install the
+      `PennyLane-Braket plugin <https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/>`__.
+      See the
+      `Catalyst configuration file <https://github.com/PennyLaneAI/catalyst/blob/main/runtime/lib/backend/openqasm/braket_local_qubit.toml>`__
+      for more details.
+
+  * - ``qrack.simulator``
+
+    - `Qrack <https://github.com/unitaryfund/qrack>`__ is a GPU-accelerated quantum computer
+      simulator with many novel optimizations including hybrid stabilizer simulation. To use this
+      device with Catalyst, make sure to install the
+      `PennyLane-Qrack plugin <https://pennylane-qrack.readthedocs.io/en/latest/>`__, and check out
+      the `QJIT compilation with Qrack and Catalyst tutorial <https://pennylane.ai/qml/demos/qrack/>`__.
+
+      See the `Catalyst configuration file <https://github.com/unitaryfund/pennylane-qrack/blob/master/pennylane_qrack/QrackDeviceConfig.toml>`__
+      for more details.
```

Comment:
I don't know if "more details" is ideal, although correct, since I would expect a different kind of information (how does it work, how do I use it, how do I install it, etc) than what would be present in a config file.
```suggestion
      for supported instructions.
```

Reply:
Up to you :)